corehq/apps/app_manager/templates/app_manager/v1/form_view_base.html

```diff
@@ -323,7 +323,7 @@
             {% trans "Edit" %}
         </button>
         {% endif %}
-        {% if allow_cloudcare %}
+        {% if can_preview_form and allow_cloudcare %}
```

Comment:
I think this should also get added to the `v2/form_view_base.html` template. Then you can run `manage.py build_app_manager_v2_diffs`